### ISSUE TITLE
Problem with jplsinfo

### ISSUE THREAD

**Issue by Jamaika1** [malaterre/charls-tools#1]
```
jplsinfo.cpp:26:12: error: no match for 'operator<<' (operand types are 'std::stringstream' {aka 'std::__cxx11::basic_stringstream<char>'} and 'const charls::spiff_profile_id')
   26 |         ss << val;
      |         ~~~^~~~~~
In file included from c:\msys1200\include\c++\12.0.1\istream:39,
                 from c:\msys1200\include\c++\12.0.1\fstream:38,
                 from jplsinfo.cpp:6:
c:\msys1200\include\c++\12.0.1\ostream:108:7: note: candidate: 'std::basic_ostream<_CharT, _Traits>::__ostream_type& std::basic_ostream<_CharT, _Traits>::operator<<(__ostream_type& (*)(__ostream_type&)) [with _CharT = char; _Traits = std::char_traits<char>; __ostream_type = std::basic_ostream<char>]'
  108 |       operator<<(__ostream_type& (*__pf)(__ostream_type&))
      |       ^~~~~~~~
c:\msys1200\include\c++\12.0.1\ostream:108:36: note:   no known conversion for argument 1 from 'const charls::spiff_profile_id' to 'std::basic_ostream<char>::__ostream_type& (*)(std::basic_ostream<char>::__ostream_type&)' {aka 'std::basic_ostream<char>& (*)(std::basic_ostream<char>&)'}
  108 |       operator<<(__ostream_type& (*__pf)(__ostream_type&))
      |                  ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
```

**Comment by malaterre** [comment]
I do not see reference to charls version. Closing as incomplete. Feel free to reopen if you are using charls 2.2.x (or up).

**Comment by Jamaika1** [comment]
I use charls https://github.com/team-charls/charls/commit/662d4f2a0238357ccc4d89cd14b1fa67d2597ff1
and boost 1.78
I don't know if the encoder works properly under Windows. Problems with help info
https://www.sendspace.com/file/1vzerw

**Comment by malaterre** [comment]
Fixed in ea442f4
